src/mono/netcore/System.Private.CoreLib/src/ILLink/ILLink.LinkAttributes.wasm.xml

```diff
@@ -116,12 +110,52 @@
       <attribute internal="RemoveAttributeInstances" />
     </type>
 
-    <!-- Microsoft.ComponentModel -->
+    <!-- System.ComponentModel -->
     <type fullname="System.ComponentModel.EditorBrowsableAttribute">
       <attribute internal="RemoveAttributeInstances" />
     </type>
   </assembly>
 
+  <!-- The following attributes are only necessary when debugging is supported -->
+  <assembly fullname="System.Private.CoreLib" feature="System.Diagnostics.Debugger.IsSupported" featurevalue="false">
```

Comment:
"featurevalue=false" means these can be removed when IsSupported is set to false, right?  Just validating it's not actually setting IsSupported to false :)

Reply:
Correct. Feature and featurevalue in the linker xml files are conditions. “Do this when this feature has this value”.